compiler/codegen/OMRAheadOfTimeCompile.hpp

```diff
@@ -94,15 +95,17 @@ class OMR_EXTENSIBLE AheadOfTimeCompile
     * it into an IteratedExternalRelocation.
     */
    static void interceptAOTRelocation(TR::ExternalRelocation *relocation) { }
+   static const size_t SIZEPOINTER = sizeof(uintptr_t);
 
-   private:
+private:
```

Comment:
i recommend protected rather than private, unless you *really* don't want subclasses to access directly.

Reply:
Hello, thank you for the review. This was just a white-space change, inside of a class that is actually used by Openj9. Will changing permissions here require changing it in inheriting class. Shall we go that route?

Reply:
ok, you can leave it as is, but please add to the list of deferred items for us to work on here

Reply:
Noted it down, thank you

Reply:
Did that, thank you!